django/contrib/admin/options.py

```diff
@@ -1072,7 +1077,7 @@ def _filter_actions_by_permissions(self, request, actions):
                 filtered_actions.append(action)
         return filtered_actions
 
-    def get_actions(self, request):
+    def get_actions(self, request, change=False):
```

Comment:
I was trying to think if we can figure out if this is coming from the change view by looking at the request (to try and reduce some of the change kwarg additions
I don't have something on the top of my head, but still thinking about it